packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -197,9 +205,9 @@ export interface IContextualMenuItem {
   href?: string;
 
   /**
-   * A collection of submenu items
+   * A collection of submenu items, or an IContextualMenuProps object to apply for the sub menu
    */
-  items?: IContextualMenuItem[];
+  items?: IContextualMenuItem[] | IContextualMenuProps;
 
```

Comment:
Just experience working on controls for OWA. We had a `Popup` control that could position itself in a variety of ways (fixed x/y, center of the screen, anchored to an element, side-docked panels, etc.) which all used different properties. It became a mess, as depending on the type you specified, some properties were silently required, while others were ignored, and it led to a lot of developer confusion and bugs.
In that case, I was able to refactor the whole thing such that the positioning logic was an object implementing an interface, and all the different strategies were implemented as their own classes with only their required props. But since breaking the API isn't an option here, I feel like overloading the type here is the best of bad options.